Makefile

```diff
@@ -28,8 +28,8 @@ build: ## Build the executable. Example: make build
 
 build-image: ## Build the images and push them to the remote registry. Example: make build-images
 	rm -rf _output
-	docker build -f Dockerfile -t $(REGISTRY)/$(REPOSITORY)/origin-projected-resource-csi-driver:$(TAG) .
-	docker push $(REGISTRY)/$(REPOSITORY)/origin-projected-resource-csi-driver:$(TAG)
+	docker build -f Dockerfile -t $(REGISTRY)/$(REPOSITORY)/origin-csi-driver-projected-resource:$(TAG) .
+	docker push $(REGISTRY)/$(REPOSITORY)/origin-csi-driver-projected-resource:$(TAG)
```

Comment:
We should let the container runtime be configurable, so folks can use `podman` without the docker alias.

Or crazy people like me, who need to do `flatpak-spawn --host podman`:

Up top:
```
CONTAINER_RUNTIME ?= docker
```
Then here:

```
$(CONTAINER_RUNTIME) build ...
$(CONTAINER_RUNTIME) push ...
```

Reply:
updated in current push